lm_eval/evaluator.py

```diff
@@ -301,7 +302,7 @@ def evaluate(
         if stderr is not None:
             results[task_prompt_name][metric + "_stderr"] = stderr(items)
             _metric_results[metric + "_stderr"] = stderr(items)
-        metric_results.append(_metric_results)
```

Comment:
Are we sure removing this line is correct?

Reply:
Thanks! No, its not. (We had commented out the bootstrapping for a bit, and it got mixed up in the mix.)